examples/Org.OData.Validation.V1.Constraint-sample.json

```diff
@@ -1,49 +1,29 @@
 {
-    "$Version": "4.0",
-    "$Reference": {
-        "https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.json": {
-            "$Include": [
-                {
-                    "$Namespace": "Org.OData.Core.V1",
-                    "$Alias": "Core"
-                }
-            ]
-        },
-        "https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Validation.V1.json": {
-            "$Include": [
-                {
-                    "$Namespace": "Org.OData.Validation.V1",
-                    "$Alias": "Validation"
-                }
-            ]
-        }
+  "$Version": "4.0",
+  "$Reference": {
+    "https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.json": {
+      "$Include": [{ "$Namespace": "Org.OData.Core.V1", "$Alias": "Core" }]
     },
-    "validation.constraint.sample": {
-        "Order": {
-            "$Kind": "EntityType",
-            "preferredDate": {
-                "$Type": "Edm.Date",
-                "$Nullable": true,
-                "@Validation.Constraint": {
-                    "FailureMessage": "Preferred date must be in the future",
-                    "Constraint": {
-                        "$Gt": [
-                            {
-                                "$Path": "preferredDate"
-                            },
-                            {
-                                "$Function": "odata.date",
-                                "$Apply": [
-                                    {
-                                        "$Function": "odata.now",
-                                        "$Apply": []
-                                    }
-                                ]
-                            }
-                        ]
-                    }
-                }
-            }
+    "https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Validation.V1.json": {
+      "$Include": [{ "$Namespace": "Org.OData.Validation.V1", "$Alias": "Validation" }]
+    }
+  },
+  "validation.constraint.sample": {
+    "Order": {
+      "$Kind": "EntityType",
+      "preferredDate": {
+        "$Type": "Edm.Date",
+        "$Nullable": true,
+        "@Validation.Constraint": {
+          "FailureMessage": "Preferred date must be in the future",
+          "Constraint": {
+            "$Gt": [
+              { "$Path": "preferredDate" },
+              { "$Function": "odata.date", "$Apply": [{ "$Function": "odata.now", "$Apply": [] }] }
```

Comment:
Too many nesting levels in one line. Has prettier got an option for restricting the number of levels?

Reply:
I couldn't find anything in https://prettier.io/docs/en/options.html.

And I don't find line 22 problematic, it is the CSDL JSON equivalent of
~~~javascript
odata.date(odata.now())
~~~

Reply:
Let the TC decide.